src/main/kotlin/racingcar/domain/RaceResult.kt

```diff
@@ -1,3 +1,25 @@
 package racingcar.domain
 
-data class RaceResult(val roundHistories: List<RoundHistory>)
+import racingcar.domain.exception.CannotFindWinnerException
+
+data class RaceResult(val roundHistories: List<RoundHistory>) {
+    fun getWinners(): List<String> {
+        val winners = getLastRoundCarInfos()
+            .groupBy { (_, position) -> position }
+            .toSortedMap().values
+            .last()
+
+        return winners.map { (name, _) -> name }
```

Comment:
RoundHistory 객체의 상태로 외부로 직접 꺼내서 처리하지 말고 메시지를 보내보면 어떨까요?

만약 메시지를 보내게된다면, 코드가 아래와 같이  바뀔 것 같아요. 🙂
```suggestion
    fun getWinners(): List<String> {
        val finalHistories = getLastRoundHistory()
        val maxPosition = finalHistories.getMaxPosition()
        return finalHistories.findSameEqualPosition(maxPosition)
```

[getter를 사용하는 대신 객체에 메시지를 보내자](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)를 읽어보시면 도움이 되실 것 같아요. 🙌